src/components/widgets/LockedContent.js

```diff
@@ -0,0 +1,219 @@
+import { useRef, useState, useEffect } from 'react';
```

Comment:
I feel like we don't need this here in webdocs. Can we instead use a local variable that refreshes on reload?

Reply:
I think the only thing is that I use browser objects like document.  

Chat GPT says: 

If you're encountering an issue where document, localStorage, and other browser-specific objects are not defined, it might be due to the fact that Next.js performs server-side rendering (SSR) during the initial page load.

In Next.js, code that relies on the browser environment should be placed inside the useEffect hook with an empty dependency array ([]). This ensures that the code is executed on the client side after the initial render, where the browser-specific objects are available.

So option 1 is - Use useEffect. 

Option 2 would be refactoring the JS to be more "React friendly". So figuring out redoing the variables and using state. 

Let me know what you think.

Reply:
Refactored the code to use local storage.